data_schema_struct.go

```diff
@@ -0,0 +1,2517 @@
+package shared
+
+type DataSchemaStruct struct {
```

Comment:
How big is this struct in memory? It seems pretty big.

Reply:
`unsafe.Sizeof`—so ignoring any slices—spit out `5120`. That seems large.

Reply:
Its a big struct but we are very far from having it filled in.